Manager/BackupManager.php

```diff
@@ -84,6 +94,10 @@ public function execute()
             return false;
         }
 
+        if ($successful) {
+            $this->eventDispatcher->dispatch(BackupCompletedEvent::EVENT_NAME, new BackupCompletedEvent());
```

Comment:
Let's do a `EverythingUploaded` (or some better name) event that is dispatched after everything got uploaded and let this generic event be dispatched after we finish everything.